config/checker/com.intellij.astub

```diff
@@ -1,12 +1,18 @@
-
 import org.checkerframework.checker.guieffect.qual.AlwaysSafe;
 import org.checkerframework.checker.guieffect.qual.SafeEffect;
 import org.checkerframework.checker.guieffect.qual.UI;
 import org.checkerframework.checker.guieffect.qual.UIEffect;
 import org.checkerframework.checker.guieffect.qual.UIPackage;
 import org.checkerframework.checker.index.qual.NonNegative;
+import com.intellij.codeInspection.util.InspectionMessage;
```

Comment:
Nit: as the powers of Spotless autoformatter can't easily reach here... pls keep the imports in abc order, in a single section